### PR TITLE
Add Consul Connect

### PR DESCRIPTION
**Pull request by daneharrigan**

### CODE REVIEW

File: hcs/swagger.json [hunk 1]
```diff
@@ -438,6 +438,10 @@
         "consulCaFile": {
           "type": "string",
           "title": "consul_ca_file is a base64-encoded ca chain file in PEM format that\nshould be used by the client agent to verify the authenticity of the\nserver"
+        },
+        "consulConnect": {
```
Comment (dynajoe) [inline hcs/swagger.json:442]: Not sure if this ship has sailed, but could this variable be named `enableConsulConnect` or `consulConnectEnabled`?

Comment (daneharrigan) [reply hcs/swagger.json:442]: Seems reasonable to me. I'll make the change

Comment (daneharrigan) [reply hcs/swagger.json:442]: Spoke with @joeandaverde and @jipperinbham. We're keeping this as-is. Azure doesn't support booleans at the moment.

**Comment by daneharrigan** [comment]
Closing this PR so I can delete the branch and create another that doesn't contain a slash in the name.

**Pull request closed without merging**